inc/authldap.class.php

```diff
@@ -1718,7 +1749,7 @@ static function searchForUsers($ds, $values, $filter, $attrs, &$limitexceeded, &
          } else {
             return false;
          }
-         if (self::isLdapPageSizeAvailable($config_ldap)) {
+         if (self::isLdapPageSizeAvailable($config_ldap) && version_compare(PHP_VERSION, '7.3') < 0) {
```

Comment:
I wonder if moving this after line 1662 is possible. Will be more readable to me.

Reply:
Well, maybe, or not; I prefer to keep code in place since I'm not sure the behavior will not change

Reply:
ok